### PR TITLE
Disarm / shove changes

### PR DESCRIPTION
**Pull request by metalgearsloth**
- Shoving just does stamina damage of 25.
- Disarm is separate RNG.

Branched from https://github.com/space-wizards/space-station-14/pull/25425

Draft to get feedback and also want to make QOL slightly better on ingame telegraphing.

:cl:
- tweak: Disarms now do stamina damage.

### CODE REVIEW

**Comment by Holinka4ever** [comment]
I have some thoughts about it to share, and they might be negative.
First of all, its really good buff for unnarmed fightings, were you can fight without relying on weapon, cause you are weapon itself.
But, in second, this giving to everyone bootleg syndicate bozing gloves, or stun baton with 9 less stamina damage, but with emp immunity. It will be funny to see that passengers shoving together sec officer, and then strip em to bone. Or several based tiders with shields tiding nuke on nuke war.
Its a double edged knife, if i understand it well.

**Comment by Hmeister-real** [comment]
I don't like how much it does, i feel like 20 would be better

**Comment by Holinka4ever** [comment]
I made little math research for this
If we add into example that tiders are showing sec officer:

1) 25 stamina damage gonna put sec into floor by 4 hits by 1 tider, or 2 hits by 2 tiders, or by 1 hit by 4 tiders.

2) 20 stamina damage gonna make sec kiss ground by 5 hits by 1 tider, or by 3 hits by 2 tiders, or by 2 hits with 3 tiders, or by 1 hit.

3)15 stamina damage gonna make sec officer look closely into tile by 7 hits by one tider, by 4 hits by 4 tiders, by 2 hits by 3 tiders, by 2 hits by 4 tiders, by 1 hit by 7 tiders.

Yeah its pretty useless information, but it gives to us thoughts that less the stamina damage, so more is teaming encourage. 
Also, if i remember well, stamina damage also giving movement speed debuff, that can also add difficulties for the victim.

Tl;dr
I would recommend turn low to 18 stamina damage, cause it will not put sec into early bed time, as 20 stamina damage do in 5 hits, the 18 damage only does in 6 hits. Also teaming with one spessmen,  allows sec officer to join list of missing in 3 hits.

Here it is.

**Comment by metalgearsloth** [comment]
Wouldn't you just use a flash or baton at that point.

**Comment by Holinka4ever** [comment]
> Wouldn't you just use a flash or baton at that point.

If in term of as sec officer, so you can run away from tiders, flashbang also can help to de-count tiders

But if in term of tider, and you got that stuff from robbering sec, i guess you dont need to shoving someone, cause you have all stuff to restrain someone in non-lethaly way with much effectivness.

Anyway, this topic about stamina damage from fists, nor about sec gear, cause the out from example.

**Comment by asperger-sind** [comment]
Imo just set it to 12 or 15 stamina damage, anything over that makes shoves a bit too good for an innate ability, this is of course assuming that shoves come off with a 100% chance now as opposed to a 25% chance.

**Comment by Hmeister-real** [comment]
Maybe combat/jackboots can make you more "steady" in combat and make shoving slightly weaker?

**Comment by K-Dynamic** [comment]
I'm more worried about crowds or groups of people shoving at once

**Comment by Holinka4ever** [comment]
> I'm more worried about crowds or groups of people shoving at once

There is nothing we can do about actually, because its human's "focus" on a one object. 
And adding mechanics to prevent these stuff, will make people really sad, because they cant shove a pep in numbers, because he has god's defence.

If you wish to make defence against numbers of shoving, guts system from fighting games may come in handy. Guts system in a short - hits due combo, does less damage than raw hit, but combo still do more damage cause it chain of hits.

But still, there is not a such a mechanic in games that dis-allows using focusing on target, cause that might de-encorage players to work together.

**Comment by Callmore** [comment]
there been much update on this?
